### PR TITLE
Client-side of auth and user modules

### PR DESCRIPTION
**Pull request by Iwasherd**
This PR is implemented client-side auth and user modules

### CODE REVIEW

File: config/auth.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const CERTIFICATE_DEVSERIAL = '00';
```
Comment (vlasenko) [inline config/auth.js]: Why do we have this setting?

Comment (Iwasherd) [reply config/auth.js]: Already removed

File: modules/user/client-react/index.tsx [hunk 2]
```diff
@@ -55,14 +67,14 @@ interface Errors {
...
-  submitting: boolean;
+  submitting?: boolean;
   errors: Errors;
   values: V;
   t: TranslateFunction;
 }
 
 const ProfileName: React.FunctionComponent<WithLogoutProps> = withLoadedUser(({ currentUser }) => (
-  <React.Fragment>{currentUser ? currentUser.fullName || currentUser.username : null}</React.Fragment>
+  <React.Fragment>{currentUser ? currentUser.profile.fullName || currentUser.username : null}</React.Fragment>
```
Comment (vsevolodbvd) [inline modules/user/client-react/index.tsx]: @Iwasherd Why did you use <React.Fragment> instead of just <>?

Comment (Iwasherd) [reply modules/user/client-react/index.tsx]: Done.

**Comment by Iwasherd** [comment]
https://github.com/sysgears/restapp/pull/9

**Pull request closed without merging**